README.md

```diff
@@ -1,28 +1,43 @@
-# CERF Global Monitoring and Alert System (GMAS)
+<img src="https://github.com/OCHA-DAP/hdx-signals/assets/HDXSignalsLogo_V2.png" width=200>
```

Comment:
This doesn't seem to be loading properly. I think you should be able to reference with markdown syntax like `![plot](assets/HDXSignalsLogo_V2.png)`

Reply:
Yeah, it doesn't yet exist until we push to main. Issue is that we don't want it to span the whole page. https://stackoverflow.com/questions/14675913/changing-image-size-in-markdown